config/software/automake.rb

```diff
@@ -15,21 +15,18 @@
 #
 
 name "automake"
-default_version "1.11.2"
+default_version "1.16"
 
 dependency "autoconf"
+dependency "perl-thread-queue"
 
 license "GPL-2.0"
 license_file "COPYING"
 skip_transitive_dependency_licensing true
 
-version "1.15" do
-  source md5: "716946a105ca228ab545fc37a70df3a3"
-end
-
-version "1.11.2" do
-  source md5: "79ad64a9f6e83ea98d6964cef8d8a0bc"
-end
+version("1.16") { source md5: "7fb7155e553dc559ac39cf525f0bb5de" }
```

Comment:
can you add this one as a sha256. We're slowly converting all the entries over to sha256

Reply:
Same of the others I added or updated?